internal/support/logging/file.go

```diff
@@ -25,23 +28,19 @@ type fileLog struct {
 	out      io.WriteCloser
 }
 
-func (fl *fileLog) closeSession() {
+func (fl *fileLog) CloseSession() {
```

Comment:
I think this code should be moved into internal/core/pkg/db/file/logging.go

Reply:
I'll need to look at this more closely.